src/triage/cli.py

```diff
@@ -254,11 +255,15 @@ def __init__(self, parser):
 
         parser.set_defaults(validate=True, validate_only=False, materialize_fromobjs=True)
 
+    def _load_config(self):
+        config_file = Store.factory(self.args.config)
+        return yaml.load(config_file.load())
```

Comment:
This _might_ be part of the solution; but, I thought the problem was that we're using `FileType` on line 172, and so argparse is attempting to load `config` from the local file system.

(Have you tested this fix?)

Reply:
You are right! I just tested  this method.... let me fix that. Thanks!